frontend/src/metabase/dashboard/components/DashboardInfoSidebar.tsx

```diff
@@ -69,7 +69,7 @@ const DashboardInfoSidebar = ({
   );
 
   return (
-    <DashboardInfoSidebarRoot>
+    <DashboardInfoSidebarRoot data-testid="sidebar-right">
```

Comment:
Can we make `DashboardInfoSidebarRoot` an HTML `aside` to keep HTML more semantic?

Reply:
Good idea. It was originally wrapped in an aside but no reason we can't just change it. Will do!